.github/workflows/gh-pages.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         env:
           CI: false
           PUBLIC_URL: "/${{ github.event.repository.name }}"
-          REACT_APP_GRAPHQL_API_ENDPOINT: "https://dubai.swayswap.io/graphql"
+          REACT_APP_GRAPHQL_API_ENDPOINT: "https://node.swayswap.io/graphql"
```

Comment:
I think it will not work for people who have opened the page before (unless everyone clear localStorage data). 

Because of this code in `App.tsx`
<img width="555" alt="image" src="https://user-images.githubusercontent.com/8636507/175074345-0eeeb799-560b-4c38-b5ca-d73e8488342d.png">

I think we should remove this code using API url saved in `localStorage`, unless I'm not seeing the use case for it

Reply:
Interesting. It's orthogonal to this PR though. Can you file an issue with your findings and fix in another PR?